### ISSUE TITLE
Getting to build with rollup (UMD format)

### ISSUE THREAD

**Issue by mikeerickson** [chalk/chalk#137]
Hey Gang

I am giving rollup a try (want to compare sizes against Browserify and Webpack)

I am receiving the follow error.  I have performed all the various fixes (including in commonJS plugin) but cant seem to make it work

Any clues?  

```
'default' is not exported by node_modules/chalk/index.js (imported by src/index.js). For help fixing this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-ex
ported-by-module
```

**Comment by sindresorhus** [comment]
The module works fine in Node.js, so Rollup is the issue tracker you want to open this on.

**Comment by mikeerickson** [comment]
I just came to you because it was saying that the module was not export default (as expected)
I did some more tweaking

- works fine with require (so I will use with require)
- if I use import * as chalk from 'chalk' it sort of works

It then throws some other bs

```
console.log(chalk.green('test'))
```

then says "green" is not exported

Guessing it is something with the import command from ES6 and how it is signed.   Also, works as import with Webpack or Browserify.

TL;DR

I changed to require and all works as expected so I will stay there for now.

**Comment by dman777** [comment]
I'm hitting this with rollup version 2.33.1. Although not the place for this, thought I would mention for anyone googling.

**Comment by Qix-** [comment]
This is a 4 year old issue, please open a ticket with Rollup if you are having issues packaging.
